packages/psmt2-frontend/psmt2-frontend.0.4.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+license: "Apache-2.0"
+synopsis: "The psmt2-frontend project"
+description: """\
+An OCaml library to parse and type-check a conservative extension of the SMT-LIB 2
+standard with prenex polymorphism.
+"""
+authors: ["Albin Coquereau"]
+maintainer: ["Albin Coquereau"]
+homepage: "https://github.com/OCamlPro-Coquera/psmt2-frontend/"
+bug-reports: "https://github.com/OCamlPro-Coquera/psmt2-frontend/issues"
+dev-repo:
+  "git+https://github.com/OCamlPro-Coquera/psmt2-frontend/tree/next.git"
+build: [
+  ["dune" "subst"] {pinned}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+depends: [
+  "ocaml" {>= "4.04.2"}
+  "dune" {>= "2.6.0"}
+  "ppx_inline_test" {with-test}
+  "ppx_expect" {with-test}
+  "odoc" {with-doc}
+  "ocamlformat" {with-test}
+  "menhir" {}
```

Comment:
Do you know if this requires a specific bound?

Reply:
Not that i know of but i will try to test older version of menhir. I'll keep you informed

Reply:
I was unable to compile the project with a version of menhir older than `20180528`. I've update the constraint, is that ok for you ?

Reply:
Perfect, thanks you very much!